# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../gmrun/gmrun-0.9.2-gcc43.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/x11-misc/gmrun/files/gmrun-0.9.2-gcc43.patch?revision=1.1

--- gmrun-0.9.2/src/gtkcompletionline.cc.orig	2003-11-16 11:55:07.000000000 +0100
+++ gmrun-0.9.2/src/gtkcompletionline.cc	2008-05-08 08:51:22.000000000 +0200
@@ -29,6 +29,7 @@
 #include <set>
 #include <sstream>
 #include <string>
+#include <cstring>
 #include <vector>
 using namespace std;
 
--- gmrun-0.9.2/src/ci_string.h.orig	2001-05-16 16:39:31.000000000 +0200
+++ gmrun-0.9.2/src/ci_string.h	2008-05-08 08:51:33.000000000 +0200
@@ -7,6 +7,7 @@
 #define __CI_STRING_H__
 
 #include <string>
+#include <cstring>
 #include <ctype.h>
 
 struct ci_char_traits : public std::char_traits<char>
--- gmrun-0.9.2/src/prefs.cc.orig	2002-08-16 12:48:22.000000000 +0200
+++ gmrun-0.9.2/src/prefs.cc	2008-05-08 08:51:54.000000000 +0200
@@ -13,6 +13,7 @@
 #include <fstream>
 #include <iostream>
 #include <stdio.h>
+#include <stdlib.h>
 
 #include <list>
 
--- gmrun-0.9.2/src/main.cc.orig	2003-11-16 11:55:07.000000000 +0100
+++ gmrun-0.9.2/src/main.cc	2008-05-08 09:03:11.000000000 +0200
@@ -14,6 +14,7 @@
 #include <gdk/gdkkeysyms.h>
 
 #include <string>
+#include <cstring>
 #include <iostream>
 #include <sstream>
 #include <vector>
